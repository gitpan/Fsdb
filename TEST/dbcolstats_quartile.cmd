prog='dbcolstats'
args='-m -q 4 midterm'
cmp='diff -c '
portable=false
subprogs=dbsort
altcmp='dbfilediff --quiet -E --exit '
altcmp_needs_input_flags=true
