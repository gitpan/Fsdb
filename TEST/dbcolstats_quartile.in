#fsdb -F S id midterm
0  51
1  24
2  23
3  44
4  75
5  61
6  30
7  43
8  49
9  73
10  47
11  44.5
12  30
13  16
14  50
15  48
16  65
17  29
18  68
19  41
20  38
21  42
22  31
23  39
24  46
