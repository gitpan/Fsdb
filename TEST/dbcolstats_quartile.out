#fsdb mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n median q1 q2 q3
44.3 15.453 34.882 6.379 37.921 50.679 0.95 1107.5 54793 16 75 25 44 31 44 50
#   | dbcolstats -m -q 4 midterm
